### DISCUSSION TITLE
How do I use tags suggestions with SpatieTagsInput ?

### DISCUSSION THREAD

**Discussion by paramwork** [filamentphp/filament#5854] [Help] [answered]
My code is -

```
SpatieTagsInput::make('tags')->type('cats')->suggestions(?)
```

How do I pull all the tags by modal or type and show in suggestions? Currently, no dropdown comes up when I click the tags input.

Thank you

**Comment by danharrin** [comment, marked as answer]
You shouldn't need to pass `suggestions()`, we do that for you already if tags with that type in the database

**Reply by paramwork** [reply]
Thanks
